### PR TITLE
update to FE doc

### PR DESCRIPTION
**Pull request by ryestew**
Ready for merging

### CODE REVIEW

File: docs/file_explorer.md [hunk 1]
```diff
@@ -1,76 +1,73 @@
...
 
-To get to the File Explorers module - click the file explorers icon.
+To get to the File Explorer module - click the File Explorer icon.
 
 ![](images/a-file-explorer1.png)
 
 File Storage
 ------------
-By default, Remix IDE stores files in **Workspaces** which are folders in your **browser's local storage**.  
+By default, Remix IDE stores files in **indexDB**.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:10]: it is `IndexedDB`

File: docs/file_explorer.md [hunk 2]
```diff
@@ -1,76 +1,73 @@
...
 File Storage
 ------------
-By default, Remix IDE stores files in **Workspaces** which are folders in your **browser's local storage**.  
+By default, Remix IDE stores files in **indexDB**.  
 
-**Important Note:** Clearing the browser storage will **permanently delete** all the files stored there. 
+Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.
+**Important Note:** Clearing the browser storage & indexDB will **permanently delete** all the files stored there.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:14]: same here

File: docs/file_explorer.md [hunk 3]
```diff
@@ -1,76 +1,73 @@
...
 
-If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix-IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix-IDE in an Electron app. 
+## File Storage outside of the browser and Version Control
 
-Workspaces
-------------
-Workspaces are new as of Remix IDE v0.11.0..  Workspaces should help clean up & organize your Files Explorer by allowing you to separate your projects.
+If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
+
+If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix-IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix-IDE in an Electron app.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:20]: I would suggest not to use `-` in `Remix-IDE`. We don't use it like that anymore

File: docs/file_explorer.md [hunk 4]
```diff
@@ -1,100 +1,101 @@
...
+To get to the File Explorer module - click the File Explorer icon.
 
 ![](images/a-file-explorer1.png)
 
+The File Explorer is for managing Workspaces and files.  There is also a context menu that pops up when you right click on a file or folder.
+
 File Storage
 ------------
-By default, Remix IDE stores files in **Workspaces** which are folders in your **browser's local storage**.  
+By default, Remix IDE stores files in **IndexDB**.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:12]: it is **IndexedDB**

File: docs/file_explorer.md [hunk 5]
```diff
@@ -1,100 +1,101 @@
...
 File Storage
 ------------
-By default, Remix IDE stores files in **Workspaces** which are folders in your **browser's local storage**.  
+By default, Remix IDE stores files in **IndexDB**.  
 
-**Important Note:** Clearing the browser storage will **permanently delete** all the files stored there. 
+Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.
+**Important Note:** Clearing the browser storage & IndexDB will **permanently delete** all the files stored there.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:16]: it is **IndexedDB**

File: docs/file_explorer.md [hunk 6]
```diff
@@ -1,100 +1,101 @@
...
+If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
 
-The Files Explorer's Workspaces all have a contracts folder, a scripts folder, a tests folder, and a README.txt.
+If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix IDE in an Electron app. 
 
-### Migrating your files to Workspaces
-If you had files stored in browser storage in versions of Remix before v0.11.0 to get these files, you will need to click on the **Migrate old filesystem to workspace** link the Home Tab.  After you click on that, you'll get a modal asking if you want to migrate and download or just migrate the files.
+Also see this article about [securing your files in Remix](https://medium.com/remix-ide/securing-you-file-in-remix-how-to-clone-and-push-f1350111aa13?source=friends_link&sk=a3dbd0d3b0b44a29a28e8c10f8821fde)
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:24]: link can be updated to https://medium.com/remix-ide/securing-you-file-in-remix-how-to-clone-and-push-f1350111aa13

Comment (ryestew) [reply docs/file_explorer.md:24]: The link I put in is the friends link so they won't be charged.  Medium has a quota of free articles - but this gets around that limitation.

**Review by Aniket-Engg — APPROVED** [review]

**Review by sullltan123 — COMMENTED** [review]
👍

**Review by sullltan123 — COMMENTED** [review]
👍

**Pull request merged**